content_sync/pipelines/concourse.py

```diff
@@ -433,7 +434,13 @@ def upsert_pipeline(self):  # pylint:disable=too-many-locals
                 web_bucket = branch_vars["publish_bucket_name"]
                 offline_bucket = branch_vars["offline_publish_bucket_name"]
                 resource_base_url = branch_vars["resource_base_url_live"]
-
+            if (
+                branch == settings.GIT_BRANCH_PREVIEW
+                or settings.ENV_NAME not in PRODUCTION_NAMES
+            ):
+                noindex = "true"
+            else:
+                noindex = "false"
```

Comment:
Rather than using `settings.ENV_NAME` here, would it be better for the env var to just be `OCW_HUGO_THEMES_NOINDEX`?

Rationale:
- that's what the thing does
- Nice not to have branch names matter
- We already have `OCW_STUDIO_ENVRIONMENT`